go/uidmap/uidmap.go

```diff
@@ -397,6 +397,30 @@ func (u *UIDMap) ClearUIDAtEldestSeqno(ctx context.Context, g libkb.UIDMapperCon
 	return nil
 }
 
+func MapUIDsReturnMap(u libkb.UIDMapper, ctx context.Context, g libkb.UIDMapperContext, uids []keybase1.UID, fullNameFreshness time.Duration, networkTimeBudget time.Duration, forceNetworkForFullNames bool) (res map[keybase1.UID]libkb.UsernamePackage, err error) {
+	var uidList []keybase1.UID
+	uidSet := map[keybase1.UID]int{}
+
+	for _, uid := range uids {
+		_, found := uidSet[uid]
+		if !found {
+			uidSet[uid] = len(uidList)
```

Comment:
What is the purpose of storing the length of uidList here?  Would map[keybase1.UID]bool be a simpler type?

Reply:
You're right - this is a leftover code after refactoring. Before there was a function in list.go that was using a map of indices to know which uid is where in uidmapper result list. Thanks!